src/gba/gba.c

```diff
@@ -595,6 +595,62 @@ void GBADebug(struct GBA* gba, uint16_t flags) {
 	gba->debugFlags = GBADebugFlagsClearSend(gba->debugFlags);
 }
 
+#ifdef USE_ELF
+bool GBAVerifyELFEntry(struct ELF* elf, uint32_t target) {
+	if (ELFEntry(elf) == target) {
+		return true;
+	}
+
+	struct ELFProgramHeaders ph;
+	ELFProgramHeadersInit(&ph, 0);
+	ELFGetProgramHeaders(elf, &ph);
+	size_t i;
+	for (i = 0; i < ELFProgramHeadersSize(&ph); ++i) {
+		Elf32_Phdr* phdr = ELFProgramHeadersGetPointer(&ph, i);
+		if (!phdr->p_filesz) {
+			continue;
+		}
+
+		size_t phdr_s = phdr->p_paddr;
+		size_t phdr_e = phdr_s + phdr->p_filesz;
+
+		// Does the segment contain our target address?
+		if ((target < phdr_s) || ((target + 4) > phdr_e)) {
```

Comment:
```suggestion
		if (target < phdr_s || target + 4 > phdr_e) {
```
That sure was a bunch of parens. Same reasoning as the previous nit cleanup; sorry I missed it that time.